index.bs

```diff
@@ -12655,8 +12725,16 @@ The \[[Prototype]] [=internal slot=] of an [=asynchronous iterator prototype obj
                 1.  Set |object|'s [=default asynchronous iterator object/is finished=] to true.
                 1.  Return [=!=] [$CreateIterResultObject$](<emu-val>undefined</emu-val>,
                     <emu-val>true</emu-val>).
-            1.  Otherwise:
+            1.  Otherwise, if the corresponding [=asynchronously iterable declaration=] is a
```

Comment:
We have `interface` defined in step 1,
> 1. Let interface be the interface for which the asynchronous iterator prototype object exists.

 could we use something like `if interface has a pair asynchronously iterable declaration` instead?